k4FWCore/scripts/k4run

```diff
@@ -267,10 +267,16 @@ def main():
         else:
             conf.setProp(optionName.rsplit(".", 1)[1], opts_dict[optionName])
 
-    if opts.verbose:
-        from Gaudi.Configuration import VERBOSE
+    if opts.log_level:
+        try:
```

Comment:
Alternatively, which levels are available can be found in 
``` python
>>> from GaudiKernel import Constants
>>> print(Constants.__all__)
['ALL',
 'VERBOSE',
 'DEBUG',
 'INFO',
 'WARNING',
 'ERROR',
 'FATAL',
 'TRUE',
 'FALSE']
```
(there is a comment about removing `TRUE` and `FALSE` since, at least, 2009...). I'm not sure if this is better than the try - except.

Reply:
> I could use the `Constants.__all__` here, but than I would still have to clean them up a bit first, I think. Or alternatively, introduce more names for the python log levels to match the ones that come from Gaudi.

As long as `LOG_LEVELS` is a subset of the constants nothing needs to be done. What I meant is simply changing the try - except to looking in `Constants.__all__` and failing if not found.